src/main/java/io/confluent/connect/jdbc/mbean/BatchMetric.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ * Copyright 2021 Swisscom (Switzerland) Ltd
+ */
+
+package io.confluent.connect.jdbc.mbean;
+
+import javax.management.InstanceAlreadyExistsException;
+import javax.management.InstanceNotFoundException;
+import javax.management.MBeanRegistrationException;
+import javax.management.MBeanServer;
+import javax.management.MalformedObjectNameException;
+import javax.management.NotCompliantMBeanException;
+import javax.management.ObjectName;
+
+import java.io.Closeable;
+import java.lang.management.ManagementFactory;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Not thread safe.
+ */
+public class BatchMetric implements BatchMetricMBean, Closeable {
+
+  private static final Logger log = LoggerFactory.getLogger(BatchMetric.class);
+  private static final MBeanServer mbs = ManagementFactory.getPlatformMBeanServer();
+  private static final String DOMAIN = "kafka.connect";
+
+  private final ObjectName name;
+
+  private Integer size;
+  private Integer position;
+
+  @Override
+  public Integer getSize() {
+    return size;
+  }
+
+  public void setSize(Integer size) {
+    this.size = size;
+  }
+
+  @Override
+  public Integer getPosition() {
+    return position;
+  }
+
+  public void setPosition(Integer position) {
+    this.position = position;
+  }
+
+  public BatchMetric(String taskId, String tableName, String batchId) {
+    log.info("Create BatchMetric with taskId={} tableName={} batchId={}",
+        taskId, tableName, batchId);
+    try {
+      Map<String, String> keyValues = new LinkedHashMap<>();
+      keyValues.put("type", "source-task-metrics");
+      keyValues.put("extension", "sbd");
```

Comment:
```suggestion
      keyValues.put("extension", "batch");
```
Or even really needed ?